cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/wizard/StepNav.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```

Comment:
nitpick: add comments for sections // React imports, // Library imports, // App imports

I noticed the same is needed for Wizard.tsx